recipes/input4mips-validation/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set name = "input4mips-validation" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/input4mips_validation-{{ version }}.tar.gz
+  sha256: a9acc322b6fd78ccb72f845d79dc2e0c885f4620c096cf6622e3ec99b9ffb5e3
+
+build:
+  entry_points:
+    - input4mips-validation = input4mips_validation.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - poetry-core
+  run:
+    - python >=3.9.0
+    - loguru >=0.7
+    - cf_xarray >=0.8
+    - xarray >=2022.0
+    - pint-xarray >=0.3
+    - numpy >=1.23, <2.0
+    - cftime >=1.6
+    - typing-extensions >=4.0
+    - pooch >=1.0
+    - netcdf4 >=1.0
+    - attrs >=22.0
+    - typer >=0.9.0
+    - validators >=0.28.3
+    - cattrs >=23.2.3
+  run_constrained:
+    - matplotlib-base >=3.0
+    - notebook >=6.0
+
+test:
+  imports:
+    - input4mips_validation
+  commands:
+    - pip check
+    - input4mips-validation --help
+  requires:
+    - pip
+
+outputs:
+  - name: input4mips-validation
+
+  # A locked version, which will help avoid installation headaches
+  # (e.g. dependency updates with breaking changes)
+  # for those who just want the CLI.
+  - name: input4mips-validation-locked
+    requirements:
+      host:
+        - python >=3.9
```

```suggestion
        - python 3.9.*
```

We probably want this here too.

---

Thanks for the tip, seems to be behaving now